styleguide/markdown-syntax-style.md

```diff
@@ -218,7 +149,7 @@ Use escaping to prevent this.
 
 ## Working with the docs site
 
-Our docs website is a custom build. This section gives some guidance on working with the setup.
+The Buildkite docs website is a custom built one. This section gives some guidance on working with the setup.
```

Comment:
Nitpick: Maybe just me but I think this reads a little more smoothly:

```suggestion
The Buildkite docs is a custom-built website. This section gives some guidance on working with the setup.
```

Reply:
Thanks, legacy content I didn't entirely see through its revision.